packages/radix-ui-themes/src/components/checkbox-group.tsx

```diff
@@ -0,0 +1,129 @@
+
+type CheckboxGroupRootElement = React.ElementRef<typeof CheckboxGroupPrimitive.Root>;
+type CheckboxGroupRootOwnProps = GetPropDefTypes<typeof checkboxGroupRootPropDefs>;
+interface CheckboxGroupRootProps
+  extends ComponentPropsWithoutColor<typeof CheckboxGroupPrimitive.Root>,
+    MarginProps,
+    CheckboxGroupRootOwnProps {}
+const CheckboxGroupRoot = React.forwardRef<CheckboxGroupRootElement, CheckboxGroupRootProps>(
+  (
+    {
+      color = checkboxGroupRootPropDefs.color.default,
+      highContrast = checkboxGroupRootPropDefs.highContrast.default,
+      size = checkboxGroupRootPropDefs.size.default,
+      variant = checkboxGroupRootPropDefs.variant.default,
+      ...props
+    }: ScopedProps<CheckboxGroupRootProps>,
+    forwardedRef
+  ) => {
+    const { __scopeCheckboxGroup, className, ...rootProps } = extractProps(props, marginPropDefs);
+    const checkboxGroupScope = useCheckboxGroupScope(__scopeCheckboxGroup);
+    return (
+      <CheckboxGroupProvider
+        scope={__scopeCheckboxGroup}
+        color={color}
+        size={size}
+        highContrast={highContrast}
+        variant={variant}
+      >
+        <CheckboxGroupPrimitive.Root
+          {...checkboxGroupScope}
+          {...rootProps}
+          ref={forwardedRef}
+          className={classNames('rt-CheckboxGroupRoot', className)}
+        />
+      </CheckboxGroupProvider>
+    );
+  }
+);
+CheckboxGroupRoot.displayName = 'CheckboxGroupRoot';
+
+type CheckboxGroupItemElement = React.ElementRef<typeof CheckboxGroupPrimitive.Item>;
+interface CheckboxGroupItemProps
+  extends Omit<React.ComponentPropsWithoutRef<typeof CheckboxGroupPrimitive.Item>, 'asChild'>,
+    MarginProps {}
+const CheckboxGroupItem = React.forwardRef<
+  CheckboxGroupItemElement,
+  ScopedProps<CheckboxGroupItemProps>
+>(({ __scopeCheckboxGroup, children, className, style, ...props }, forwardedRef) => {
+  const context = useCheckboxGroupContext('item', __scopeCheckboxGroup);
+  const checkboxGroupScope = useCheckboxGroupScope(__scopeCheckboxGroup);
+  const { className: checkboxClassName } = extractProps(
+    context,
+    checkboxGroupRootPropDefs,
+    marginPropDefs
+  );
+  // Render `<Text as="label" size={context.size}>` if children are provided, otherwise
+  // render a simple `<label>` to avoid setting a height / line height on the element.
+  // This is so that you can easily use this component in own layout.
+  const Label = children ? Text : 'label';
+  const labelProps = children ? { as: 'label' as const, size: context.size } : {};
+  return (
+    <Label {...labelProps} className={classNames('rt-CheckboxGroupItem', className)} style={style}>
+      <CheckboxGroupPrimitive.Item
+        data-accent-color={context.color}
+        {...checkboxGroupScope}
+        {...props}
+        ref={forwardedRef}
+        className={classNames(
+          'rt-reset',
+          'rt-BaseCheckboxRoot',
+          'rt-CheckboxRoot',
```

Comment:
idk how we feel about using `'rt-CheckboxRoot'` directly—alternatively, copy-paste the focus styles?

Reply:
Duplicated the focus and disabled styles